airbyte-integrations/connectors/source-sftp-bulk/source_sftp_bulk/client.py

```diff
@@ -168,15 +169,22 @@ def get_files(self, prefix, search_pattern=None, modified_since=None, most_recen
 
         return sorted_files
 
+    def peek_line(self, f):
+        pos = f.tell()
+        line = f.readline()
+        f.seek(pos)
+        return line
+
     @backoff.on_exception(backoff.expo, (socket.timeout), max_tries=5, factor=2)
     def fetch_file(self, fn: Mapping[str, Any], file_type="csv") -> pd.DataFrame:
```

Comment:
Is it possible to add a parameter for separator, so user will have more control over the separator. If there is no separator passed, then Sniffer can be used.